circleci/docker-publish

```diff
@@ -98,7 +100,8 @@ fi
 docker login -u $DOCKER_USER -p $DOCKER_PASS --email="$DOCKER_EMAIL" || docker login -u $DOCKER_USER -p $DOCKER_PASS
 if [ "$ECR_PUSH_ID" != "" ]; then
   echo "ECR access_id available. Logging into ECR..."
-  ecr_login
+  ecr_login $ECR_REGION
+  ecr_login us-west-2
```

Comment:
Maybe it'd be clearer to hardcode at the top of this script two variables like `ECR_REGION_{PRIMARY,SECONDARY}` and make a clean split from the environment config in circle

Reply:
Fair point, I'll do that.